tests/testthat/test-helpers_knitr.R

```diff
@@ -99,20 +96,31 @@ test_that("asis parameter works correctly for all implemented methods", {
       obj <- do.call(paste0(".Default", cls), list())
       # If the default knit_print method has been overridden, test it
       if (h_custom_method_exists(knit_print, obj)) {
-        rv <- knit_print(obj)
         # Default behaviour
+        rv <- knit_print(obj)
+        if (is.null(rv)) print(paste0("asis = TRUE [1]: NULL value for class ", cls, "."))
```

Comment:
any chance to rewrite the message to something more easily understood by future developers?

Reply:
e.g. "knit_print method does not work bla bla for class foo"?

Reply:
Done.